conf/config.json

```diff
@@ -1,7 +1,7 @@
 {
   "system": {
     "updatechecker": false,
-    "memcache.local": "\\OC\\Memcache\\APCu",
+    "memcache.local": "\\OC\\Memcache\\Redis",

```

Comment:
Hmm why did you change this value?

Reply:
i had a failed install with a warning about something involving APCu (sorry I can't recall the specific warning).
I am reloading a test with APCu set

Reply:
It could be related to https://github.com/YunoHost-Apps/nextcloud_ynh/issues/419

I have to admit that I really don't know what is the best option here.

Reply:
I found a topic on the subject on Nextcloud forum. Trying to find it...

Reply:
The CI warning:
`141712 INFO DEBUG - OC\HintException: [0]: Memcache \OC\Memcache\APCu not available for local cache (Is the matching PHP module installed and enabled?)`

The issue I found on Nextcloud forum:
https://help.nextcloud.com/t/solved-apcu-caching-gives-error/9403/5

but to be fair I don't understand either what the implications are... `¯\_ (ツ) _/¯`

Reply:
https://docs.nextcloud.com/server/latest/admin_manual/configuration_server/caching_configuration.html#id1

We need to install a package, and enable it in the [php conf file](https://github.com/YunoHost-Apps/nextcloud_ynh/blob/testing/conf/extra_php-fpm.conf)

so it's related to #419 I'll fix that later (how it worked before O.O)

Reply:
Will it make sense to only use Redis for memory caching?  it seems now that we are using APCu and Redis (cf. config.json)

Reply:
According to the doc: 

> APCu is faster at local caching than Redis. If you have enough memory, use APCu for Memory Caching and Redis for File Locking. If you are low on memory, use Redis for both.

Reply:
It's not a bug, it's a feature:

https://github.com/nextcloud/server/issues/27781#issuecomment-873543420